codegen/smithy-aws-swift-codegen/src/main/kotlin/software/amazon/smithy/aws/swift/codegen/customization/route53/StripHostedZoneURLPathMiddleware.kt

```diff
@@ -1,55 +0,0 @@
-package software.amazon.smithy.aws.swift.codegen.customization.route53
-
-import software.amazon.smithy.codegen.core.Symbol
-import software.amazon.smithy.model.shapes.OperationShape
-import software.amazon.smithy.swift.codegen.Middleware
-import software.amazon.smithy.swift.codegen.MiddlewareGenerator
-import software.amazon.smithy.swift.codegen.SwiftDependency
-import software.amazon.smithy.swift.codegen.SwiftWriter
-import software.amazon.smithy.swift.codegen.integration.ProtocolGenerator
-import software.amazon.smithy.swift.codegen.integration.middlewares.handlers.MiddlewareShapeUtils
-import software.amazon.smithy.swift.codegen.integration.steps.OperationInitializeStep
-
-class StripHostedZoneURLPathMiddleware(
```

Comment:
This middleware is no longer rendered since a reusable one in `AWSClientRuntime` is used instead.